products/BellHybrid/services/audio/VolumeFade.cpp

```diff
@@ -0,0 +1,125 @@
+// Copyright (c) 2017-2024, Mudita Sp. z.o.o. All rights reserved.
+// For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
+
+#include "VolumeFade.hpp"
+#include <log/log.hpp>
+
+namespace audio
+{
+    namespace
+    {
+        constexpr auto timerName{"volumeFadeTimer"};
+        constexpr std::chrono::milliseconds fadeInterval{33};
```

Comment:
```suggestion
        using namespace std::chrono_literals;
        constexpr auto fadeInterval{33ms};
```
It's easier to read and should be functionally equivalent :wink:

Reply:
done